### PR TITLE
finish UI for resolving player tags

### PR DESCRIPTION
**Pull request by yedi**
@jhertz @jiangtyd @garsh0p

### CODE REVIEW

File: model.py [hunk 1]
```diff
@@ -339,6 +339,12 @@ def set_alias_id_mapping(self, alias, id):
             'player_alias': alias,
             'player_id': id
         })
+
+    def delete_alias_id_mapping(self, alias):
+        for mapping in self.alias_to_id_map:
+            if mapping['player_alias'] == alias:
+                self.alias_to_id_map.remove(mapping)
+                return mapping
```
Comment (garsh0p) [inline model.py]: Can you explicitly `return None` outside of the loop in case we don't find a match?

**Comment by garsh0p** [comment]
The approach here isn't what I was envisioning. I was thinking about a single PUT to /tournaments that would pass and set the entire alias -> ID mapping (based on the state of the checkboxes on the clientside).

**Comment by yedi** [comment]
wouldn't that method be a lil worse for usability? Since the submit tournament button can only be pressed once, it forces the user to make all the changes then submitting the tournament on that page without ever browsing away from that page.

An admin couldn't like start work on a tournament and then come back to it later because all their changes would get cleared when they reload the page. It also doesn't allow for multiple admins to work on aliases together. Imagine you wanted to send the page to a few other admins just to confirm that all the aliases / tags were as expected, or ask them to fill in any of the remaining tags, you wouldn't be able to do this if you never persist the checkboxes. (This is actually pretty similar to how Vestboy and me worked to generate aliases for the last ranking, just with collabedit pads instead)

What you suggest is definitely less work though -- lmk what you think of what I mentioned

**Comment by garsh0p** [comment]
Well, there are 2 different concepts here:

1) the PUT method to update the underlying PendingTournament database object. This would NOT convert the PendingTournament to a real Tournament
2) some other way to actually do the finalization/conversion

I prefer the PUT method since it makes the code a lot easier and more intuitive from an API perspective. You can get the same functionality by simplying PUTing every time someone ticks a checkbox (obviously less efficient since you're passing a larger payload every time but we can worry about efficiency later).

**Comment by yedi** [comment]
ah, i feel you. Sounds good I'll move to a PUT for that. I'll also add the finalization to my list of todos and get to that as well

**Comment by yedi** [comment]
updated

File: model.py (hunk 1 above)
Comment (jiangtyd) [inline model.py]: i wanna refactor this to actually be a map inside the `PendingTournament` object, but I guess that can be a separate PR

**Comment by yedi** [comment]
The UI (button) for doing the finalization / conversion of a pending tournament to real a tournament is already there. Just need to wait for Damien's PR to get pulled so that I can hook up that button to the endpoint he created.
